jetty-test-helper/src/main/java/org/eclipse/jetty/toolchain/test/jupiter/TestTrackerExtension.java

```diff
@@ -0,0 +1,41 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2018 Mort Bay Consulting Pty. Ltd.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.toolchain.test.jupiter;
+
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.lang.reflect.Method;
+import java.util.Optional;
+
+public class TestTrackerExtension implements BeforeEachCallback
+{
+
+    @Override
+    public void beforeEach( ExtensionContext extensionContext )
+    {
+        //String displayName = extensionContext.getDisplayName();
+        // we do not manage displayName here
+        Optional<Method> method = extensionContext.getTestMethod();
+        Optional<Class<?>> clazz = extensionContext.getTestClass();
+        System.err.printf("Running %s.%s()%n",
+                          clazz.isPresent()?clazz.get().getName():"",
+                          method.isPresent()?method.get().getName():"");
```

Comment:
We also need to print out any parameters the test is run with.   Can't we just use the display name, which I think already includes that?

Reply:
not exactly. depending on the context (method with or without parameters) displayName can have a different value.
Method without parameters: it contains method name.
Method with parameters: it contains the parameters value.
parameters are not (yet) available (https://github.com/junit-team/junit5/issues/1139)
Until this one is fixed we can use displayName for method with parameters.